Sources/StreamChatUI/Utils/UIImageView+Extensions.swift

```diff
@@ -64,11 +64,7 @@ extension UIImageView {
         
         let imageKey = components.imageCDN.cachingKey(forImage: url)
         let urlRequest = components.imageCDN.urlRequest(forImage: url)
-        let request = ImageRequest(
-            urlRequest: urlRequest,
-            processors: preprocessors,
-            options: ImageRequestOptions(filteredURL: imageKey)
-        )
+        let request = ImageRequest(urlRequest: urlRequest, processors: preprocessors, userInfo: [.imageIdKey: imageKey])
```

Comment:
@TheCoordinator this is breaking the Sample app + UI Tests. Do you have time to make the change, or want me to take control?

Reply:
@nuno-vieira Please take control :-)